apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx

```diff
@@ -43,10 +44,18 @@ const styles = {
     whiteSpace: 'pre-wrap'
   },
   inputParent: {
-    display: 'flex'
+    display: 'flex',
+    alignItems: 'baseline'
   },
   add: {
-    margin: DEFAULT_MARGIN
+    margin: DEFAULT_MARGIN,
+    color: color.dark_charcoal,
+    borderColor: color.dark_charcoal,
+    ':disabled': {
```

Comment:
NIT (up to you): if you change the add button to a `<Button>`, you shouldn't need the `disabled` styles.